viewmaster/viewmaster.py

```
@@ -260,128 +333,109 @@ def validate_holdings_paths(abspaths):
 
-        if os.path.islink(symlink):         # exists and is a symlink
-            try:
-                os.remove(symlink)
-            except OSError:
-                LOGGER.error('Cannot remove outdated symlink', symlink)
+    global HOLDINGS_PATHS
+
+    try:
+        paths = get_holdings_paths()
+        paths = validate_holdings_paths(paths, logger)
+    except Exception as e:
```

Ruff (0.14.11)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)